cloud/services/compute/instancegroup.go

```diff
@@ -60,10 +60,12 @@ func (s *Service) DeleteInstanceGroups() error {
 		name := path.Base(groupSelfLink)
 		op, err := s.instancegroups.Delete(s.scope.Project(), zone, name).Do()
 		if err != nil {
-			return errors.Wrapf(err, "failed to create backend service")
+			if !gcperrors.IsNotFound(err) {
```

Comment:
Similar to above, this conditional can be combined with the `err != nil` check above